### ISSUE TITLE
[Accessibility] ExtendedPeoplePicker has list role but no child with listitem roles on initialisation

### ISSUE THREAD

**Issue by thuy-n-lu** [microsoft/fluentui#15982]
### Environment Information

- **Package version(s)**: 7.148.2
- **Browser and OS versions**: 87.0.637.0 (Official build) dev (64-bit)
Windows OS version: 2004 (Build 20211.1000)

### Describe the issue:

Role for the ExtendedPeoplePicker input is defined as 'list' but the required ARIA child role not present: list item when initialised.
### Please provide a reproduction of the issue in a codepen:
https://fabricweb.z5.web.core.windows.net/oufr/7.148.2/#/examples/extendedpeoplepicker
#### Actual behavior:

Role for the ExtendedPeoplePicker input is defined as 'list' but the required ARIA child role not present: list item.

#### Expected behavior:

Elements with an ARIA role List should have child listitem

### Documentation describing expected behavior

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/List_role

**Comment by paulgildea** [comment]
@thuy-n-lu Thanks for filing this issue. We'll look in to it.
